Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/InvoluntaryProject/Summary.cshtml

```diff
@@ -130,6 +130,7 @@
                         PFI (private finance initiative)
                     </dt>
                     <dd class="govuk-summary-list__value">@(Model.Establishment?.ViewAcademyConversion?.Pfi ?? "Does not apply")</dd>
+                    <dd class="govuk-summary-list__value"></dd>
```

Comment:
Why don't we need this value anymore?

Reply:
ah, it was to fix this gap.
![image](https://user-images.githubusercontent.com/109524605/219426692-64a2c90e-c164-491a-b69c-da5827442a59.png)